src/mobile/apps/artwork/components/related_artworks/index.coffee

```diff
@@ -6,8 +6,8 @@ metaphysics = require '../../../../../lib/metaphysics.coffee'
 { displayRelatedWorks } = require './display_related_works.coffee'
 
 module.exports = ->
-  isArtist = false
-  isPartner = false
```

Comment:
We used to have special logic for `isArtist` and `isPartner`, because those were the only sections which included a link to "view all works". Since we have this for _all_ sections except the "related works" section now, I updated the logic accordingly.